Crash when calling NotificationChannelGroupCompat.isEnabled on Oreo device

### Steps to reproduce problem: 
1. In NotificationChannelCompat.applyChannel (line 213), change the `group.isBlocked()` check to `!group.isEnabled()`.
1. Install, run example app, and hit Notify button.

### What happened:
Crash from calling NotificationChannelGroup.isBlocked() on pre API 28 device.
```
     Caused by: java.lang.NoSuchMethodError: No virtual method isBlocked()Z in class Landroid/app/NotificationChannelGroup; or its super classes (declaration of 'android.app.NotificationChannelGroup' appears in /system/framework/framework.jar)
        at com.lionscribe.open.notificationchannelcompat.NotificationChannelGroupCompat.isEnabled(NotificationChannelGroupCompat.java:235)
        at com.lionscribe.open.notificationchannelcompat.NotificationChannelCompat.applyChannel(NotificationChannelCompat.java:213)
        at com.lionscribe.open.notificationchannelcompat.example.MainActivity.doNotify(MainActivity.java:109)
        at com.lionscribe.open.notificationchannelcompat.example.MainActivity.onClick(MainActivity.java:63)
```

### What should happen:
NotificationChannelGroup.isBlocked() should be equivalent to calling !NotificationChannelGroup.isEnabled(). This is just missing from the fix for issue #1.

--- a/notificationchannelcompat/src/main/java/com/lionscribe/open/notificationchannelcompat/NotificationChannelGroupCompat.java
+++ b/notificationchannelcompat/src/main/java/com/lionscribe/open/notificationchannelcompat/NotificationChannelGroupCompat.java
@@ -216,12 +216,7 @@ public final class NotificationChannelGroupCompat implements Parcelable {
      * {@link NotificationChannel#getImportance()}.
      */
     public boolean isBlocked() {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-            return _notificationChannelGroup.isBlocked();
-        } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            return false; // not an option
-        }
-        return !mEnabled;
+        return !isEnabled();
     }
 
     /**
@@ -231,8 +226,10 @@ public final class NotificationChannelGroupCompat implements Parcelable {
      * {@link NotificationChannel#getImportance()}.
      */
     public boolean isEnabled() {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
             return !_notificationChannelGroup.isBlocked();
+        } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            return true; // not an option.
         }
         return mEnabled;
     }

